cfme/tests/physical_infrastructure/test_physical_rack_details.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
+def physical_rack(appliance, provider, setup_provider_modscope):
+    # Get and return the first physical rack
+    physical_racks = appliance.collections.physical_racks.all(provider)
```

Comment:
all() does not receive provider, so there's no need to pass it here or receive it as a fixture here